lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery/Sequential.hs

```diff
@@ -331,14 +333,13 @@ accXPubTxt xpub =
     in T.take 8 keyFormatted <> "..." <> T.takeEnd 8 keyFormatted
 
 instance Buildable ScriptTemplate where
-    build (ScriptTemplate cosignersMap script') = mempty <>
+    build (ScriptTemplate cosignersMap script') =
         "Cosigners:" <> build (presentCosigners cosignersMap) <>
         " Script:" <> build (T.pack (show script'))
       where
-        printCosigner (Cosigner ix) =
-            "cosigner#"<> T.pack (show ix)
-        presentCosigners =
-            Map.foldrWithKey (\c k acc -> acc <> "| " <> printCosigner c <> " " <> accXPubTxt k ) mempty
+        printCosigner (Cosigner ix) = "cosigner#" <> T.pack (show ix)
+        presentCosigners = mempty & Map.foldrWithKey \c k acc ->
+            acc <> "| " <> printCosigner c <> " " <> accXPubTxt k
```

Comment:
```suggestion
        presentCosigners =
             Map.foldrWithKey (\c k acc -> acc <> "| " <> printCosigner c <> " " <> accXPubTxt k) mempty
```

The previous version is easier to read?

I would also be ok with
```
             flip Map.foldrWithKey mempty $ \c k acc ->
                 acc <> "| " <> printCosigner c <> " " <> accXPubTxt k
```
if I really wanted to have the function as the last argument.

Reply:
I'll take your suggestion as I want to have the function as a last argument - this way the function could be split into several lines nicely.